test/BUILD.gn

```diff
@@ -283,6 +284,7 @@ test("brave_browser_tests") {
     "//brave/browser/extensions/brave_extension_functional_test.h",
     "//brave/browser/extensions/api/brave_shields_api_browsertest.cc",
     "//brave/browser/extensions/api/brave_theme_api_browsertest.cc",
+    "//brave/browser/net/brave_network_delegate_base_browsertest.cc",
```

Comment:
maybe merge this and the next one? Or give more descriptive names to source files: this one against hsts, etc